src/lib/run-dev.js

```diff
@@ -365,7 +365,7 @@ async function invokeAction ({ actionRequestContext, logger }) {
   if (actionFunction) {
     try {
       process.chdir(path.dirname(action.function))
-      process.env.__OW_ACTION_NAME = actionName
+      process.env.__OW_ACTION_NAME = `/${process.env.__OW_NAMESPACE}/${packageName}/${actionName}`
```

Comment:
is it possible that `$packageName` can be undefined? Should we check for that?

Reply:
Things will break long before it gets here if packageName is not defined.
See `serveWebAction` #L471